### PR TITLE
feat(3000): Add more sidebars

### PR DESCRIPTION
**Pull request by Twixes**
## Changes

This adds sidebars for a few of the remaining PostHog 3000 navbar items:
- cohorts
- insights (no "Load more yet" though)
- persons & groups (no "Load more yet" though)

### CODE REVIEW

File: frontend/src/scenes/groups/groupsList.test.ts
Comment (Twixes) [inline frontend/src/scenes/groups/groupsList.test.ts:1]: This suite only tested things that shouldn't actually be the responsibility of the logic. It should be handled at the scene system level – which is the case now.

File: frontend/src/scenes/groups/groupsListLogic.ts [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
     results: Group[]
 }
 
+export interface GroupsListLogicProps {
+    groupTypeIndex: number
+}
+
 export const groupsListLogic = kea<groupsListLogicType>({
+    props: {} as GroupsListLogicProps,
+    key: (props: GroupsListLogicProps) => props.groupTypeIndex,
```
Comment (Twixes) [inline frontend/src/scenes/groups/groupsListLogic.ts:23]: Previously there could only be one instance of `groupsListLogic` globally, so we couldn't load multiple group types at once. Now that this is keyed, we can

**Review by thmsobrmlr — CHANGES_REQUESTED** [review]
Switching between Cohorts and People & Groups leads to an unmounted logic issue
https://github.com/PostHog/posthog/assets/1851359/4d0208d9-ff7d-49a6-b443-1a97875a5740

**Comment by Twixes** [comment]
Thanks for catching, that is resolved now!

File: frontend/src/layout/navigation-3000/navbarItems.tsx [hunk 2]
```diff
@@ -27,18 +30,46 @@ export const NAVBAR_ITEMS: NavbarItem[][] = [
...
+            identifier: Scene.Annotations,
+            label: 'Annotations',
+            icon: <IconComment />,
+        },
+    ],
+    [
         {
             identifier: Scene.SavedInsights,
-            label: 'Insights',
+            label: 'Analytics',
```
Comment (thmsobrmlr) [inline frontend/src/layout/navigation-3000/navbarItems.tsx:70]: That name is way more understandable. We should also change the name throughout the rest of the UI at some point.

**Review by thmsobrmlr — APPROVED** [review]
Looks good and soon we'll have the new navigation finished!

A couple of things I noticed (in order of what I think is more important):
- One thing I'd like to change is that the groups should be displayed by name and not by id, as "Hedgebox Inc." will mean more to the user than "01879987-a532-0000-967f-4503bf0fc283".
- Seems there is a small styling regression - see the frontend/__snapshots__/posthog-3000-sidebar--feature-flags.png snapshot.
- The persons endpoint is quite slow for me and I see the loading spinner every time I switch tabs - should probably have a more lightweight search and/or caching here.

**Comment by Twixes** [comment]
Very good points, added support for group names and fixed the styling issues. As for persons loading, it's a general performance issue I suppose.

**Pull request merged**